### ISSUE TITLE
I can't log in

### ISSUE THREAD

**Issue by allanpedroso** [Sjord/growatt_api_client#19]
even by putting username and password in single quotes the login exception is raised

**Comment by Sjord** [comment]
Thank you for the feedback. I am sorry to hear that you have problems logging in. Unfortunately, the server doesn't give much information on what goes wrong, so this is a little difficult to troubleshoot. The most common issue is that the username and password are incorrect. Can you log in to the web portal with the same credentials? Can you try modifying [__main__.py](https://github.com/Sjord/growatt_api_client/blob/master/growatt/__main__.py) and putting your username and password directly in there? E.g.

```python
username = "allan"
password = "something"
```

**Comment by allanpedroso** [comment]
yes, in the portal i can login with the same data. The problem persists even if I pass the parameters directly to main.py :((

**Comment by Sjord** [comment]
Hmm, I am not sure what is going on in that case.

**Comment by Sjord** [comment]
Do you perhaps have an account [on another server](https://github.com/Sjord/growatt_api_client/issues/20)? Where do you normally log in?
